marshal/consumer_test.go

```diff
@@ -125,6 +125,12 @@ func (s *ConsumerSuite) Produce(topicName string, partID int, msgs ...string) in
 	return offset
 }
 
+func (s *ConsumerSuite) WaitForRsteps(c *C, cluster *KafkaCluster, numSteps int) {
```

Comment:
Did you want something similar for those waitForRsteps in marshal/claim_test.go?

Reply:
Yeah I ended up doing it 2 different ways but should probably standardize.  I think I'll remove the helper functions since we usually don't mind doing a bunch of error checking copy/pasting in tests

Reply:
jk I'll keep the helper functions